contracts/MultiPartyEscrow.sol

```diff
@@ -31,11 +32,14 @@ contract MultiPartyEscrow {
  
     ERC20 public token; // Address of token contract
     
-    //TODO: optimize events. Do we need more (or less) events?
-    event EventChannelOpen       (uint256 channelId,         address indexed sender, address indexed recipient, uint256 indexed groupId);
-    //event EventChannelReopen     (uint256 channelId,         address indexed sender, address indexed recipient, uint256 indexed groupId, uint256 nonce);
-    //event EventChannelTorecipient(uint256 indexed channelId, address indexed sender, address indexed recipient, uint256 amount);
-    //event EventChannelTosender   (uint256 indexed channelId, address indexed sender, address indexed recipient, uint256 amount);
+
+    // Events
+    event ChannelOpen(uint256 channelId, address indexed sender, address indexed recipient, uint256 indexed groupId, address signer, uint256 amount, uint256 expiration);
+    event ChannelClaim(uint256 indexed channelId, address indexed recipient, uint256 claimAmount, uint256 sendBackAmount, uint256 keepAmpount);
+    event SenderClaim(uint256 indexed channelId, uint256 claimAmount);
+    event ChannelExtend(uint256 indexed channelId, uint256 newExpiration);
+    event AddFunds(uint256 indexed channelId, uint256 newFunds);
```

Comment:
Sorry. I haven't seen it first time. 
I would also suggest to rename 
* SenderClaim to ChannelSenderClaim
* AddFunds to ChannelAddFunds

Like this we will have all events related to channels have prefix Channel.

Reply:
Thought of keeping the event names as simple... Updated as per your comments.